config.yml

```diff
@@ -7723,3 +8036,2703 @@ production_recipes:
+      Efficiency 4:
+        type: DIG_SPEED
+        level: 4
+        probability: 0.1
+      Unbreaking 3:
+        type: DURABILITY
+        level: 3
+        probability: 0.08
+      Fortune 3:
+        type: LOOT_BONUS_BLOCKS
+        level: 3
+        probability: 0.42
+      Silk Touch 1:
+        type: SILK_TOUCH
+        level: 1
+        probability: 0.02
+      Efficiency 3:
+        type: DIG_SPEED
+        level: 3
+        probability: 0.02
+      Fortune 2:
+        type: LOOT_BONUS_BLOCKS
+        level: 2
+        probability: 0.36
+  Create_Wooden_Spade:
+    name: Create Wooded Spade
+    production_time: 40
+    inputs:
+      Wooden Spade:
+        material: WOOD_SPADE
+        amount: 1
+      Emerald Blocks:
+        material: EMERALD_BLOCK
+        amount: 2
+    outputs:
+      Enchanted Wooden Spade:
+        material: WOOD_SPADE
+        amount: 1
+    enchantments:
+      Efficiency 4:
+        type: DIG_SPEED
+        level: 4
+        probability: 0.1
+      Unbreaking 3:
+        type: DURABILITY
+        level: 3
+        probability: 0.08
+      Fortune 3:
+        type: LOOT_BONUS_BLOCKS
+        level: 3
+        probability: 0.42
+      Silk Touch 1:
+        type: SILK_TOUCH
+        level: 1
+        probability: 0.02
+      Efficiency 3:
+        type: DIG_SPEED
+        level: 3
+        probability: 0.02
+      Fortune 2:
+        type: LOOT_BONUS_BLOCKS
+        level: 2
+        probability: 0.36
+  Create_Wooden_Axe:
+    name: Create Wooden Axe
+    production_time: 40
+    inputs:
+      Wooden Axe:
+        material: WOOD_AXE
+        amount: 1
+      Emerald Blocks:
+        material: EMERALD_BLOCK
+        amount: 2
+    outputs:
+      Enchanted Wooden Axe:
+        material: WOOD_AXE
+        amount: 1
+    enchantments:
+      Efficiency 4:
+        type: DIG_SPEED
+        level: 4
+        probability: 0.1
+      Unbreaking 3:
+        type: DURABILITY
+        level: 3
+        probability: 0.08
+      Fortune 3:
+        type: LOOT_BONUS_BLOCKS
+        level: 3
+        probability: 0.42
+      Silk Touch 1:
+        type: SILK_TOUCH
+        level: 1
+        probability: 0.02
+      Efficiency 3:
+        type: DIG_SPEED
+        level: 3
+        probability: 0.02
+      Fortune 2:
+        type: LOOT_BONUS_BLOCKS
+        level: 2
+        probability: 0.36
+  Create_Fishing_Rod:
+    name: Create Fishing Rod
+    production_time: 40
+    inputs:
+      Fishing Rod:
+        material: FISHING_ROD
+        amount: 1
+      Emerald Blocks:
+        material: EMERALD_BLOCK
+        amount: 1
+    outputs:
+      Enchanted Fishing Rod:
+        material: FISHING_ROD
+        amount: 1
+    enchantments:
+      Unbreaking 3:
+        type: DURABILITY
+        level: 3
+        probability: 0.16
+      Efficiency 4:
+        type: DIG_SPEED
+        level: 4
+        probability: 0.16
+      Efficiency 4:
+        type: DIG_SPEED
+        level: 4
+        probability: 0.06

```

Comment:
You have Efficiency 4 three times here; I'm guessing this was a TODO: to replace with new enchantment types but was never done?